packages/@aws-cdk/aws-ec2/lib/bastion-host.ts

```diff
@@ -80,6 +81,22 @@ export interface BastionHostLinuxProps {
    * @default - Uses the block device mapping of the AMI
    */
   readonly blockDevices?: BlockDevice[];
+
+  /**
+   * Apply the given CloudFormation Init configuration to the instance at startup
+   *
+   * @default - no CloudFormation init
+   */
+  readonly init?: CloudFormationInit;
+
+  /**
+    * Use the given options for applying CloudFormation Init
+    *
+    * Describes the configsets to use and the timeout to wait
+    *
+    * @default - default options
+    */
```

Comment:
Super small nitpick; the spacing here is off (indented one space too many).

Reply:
Fixed.